kalite/coachreports/static/js/coachreports/coach_reports/models.js

```diff
@@ -0,0 +1,32 @@
+var StateModel = Backbone.Model.extend({
+    defaults: {
+        group_id: window.GROUP_ID || "",
+        facility_id: window.FACILITY_ID || ""
+    }
+});
+
+var FacilityModel = Backbone.Model.extend();
+
+var GroupModel = Backbone.Model.extend();
+
+var FacilityCollection = Backbone.Collection.extend({
+    url: FACILITY_RESOURCE_URL
+});
+
+var GroupCollection = Backbone.Collection.extend({
+    url: GROUP_RESOURCE_URL
```

Comment:
It is. Unfortunately, these are TastyPie URLs which do not play nicely with django-reverse-js :(

@jamalex Another argument for Django RESTful (assuming it plays nicely with Django-reverse-js).